pkg/controller/application/application_controller_test.go

```diff
@@ -542,6 +573,10 @@ func TestDeletingAbortedReleases(t *testing.T) {
 			Status:  corev1.ConditionFalse,
 			Message: fmt.Sprintf(ReleaseActiveMessageFormat, releaseBar.Name),
 		},
+		{
+			Type:	shipper.ApplicationConditionTypeRolloutBlock,
+			Status: corev1.ConditionFalse,
```

Comment:
None of the test changes include enabled rollout block. I believe it might be a good idea to cover this case as well.

Reply:
I separated the tests I wrote in a different pr: https://github.com/bookingcom/shipper/pull/119 
this is here just so the current tests will not fail :)

Reply:
Oh, that totally makes sense, thanks! Sorry for some extra noise!